README.md

```diff
@@ -51,3 +51,7 @@ gulp
       * fetch.user.report.error - logged when encountered problems during search for user reports
       * fetch.user.error - logged when encountered error during user fetch
 
+### Status
+
+  * `Settings are empty` - `error` - returned when we have no script code saved in the settings
+  * `Settings are referencing invalid values` - `error` - returned when we have a script with syntax error
```

Comment:
I check two things now:

1. is it empty -> error
2. is it incorrect JS syntax -> error (without running the code)